.cirrus.yml

```diff
@@ -27,7 +27,7 @@ task:
     image_project: pg-ci-images
     # NetBSD version should match
     # https://github.com/anarazel/pg-vm-images/blob/main/packer/netbsd.pkrvars.hcl
-    image: family/pg-ci-netbsd-vanilla-9-3
+    image: family/pg-ci-netbsd-vanilla-10-0
     platform: netbsd
   install_script:
     - export PKG_PATH="http://cdn.NetBSD.org/pub/pkgsrc/packages/NetBSD/$(uname -p)/$(uname -r|cut -f '1 2' -d.)/All/"
```

Comment:
This seems to be the offending line for the NetBSD failures like in https://github.com/haskell/bytestring/pull/673/checks?check_run_id=27752349204

If you see in in the log

```txt
pkg_add: Warning: package `git-2.45.2' was built for a platform:
pkg_add: NetBSD/x86_64 9.0 (pkg) vs. NetBSD/x86_64 9.3 (this host)
pkg_add: Warning: package `git-base-2.45.2' was built for a platform:
pkg_add: NetBSD/x86_64 9.0 (pkg) vs. NetBSD/x86_64 9.3 (this host)
pkg_add: Warning: package `pcre2-10.44' was built for a platform:
pkg_add: NetBSD/x86_64 9.0 (pkg) vs. NetBSD/x86_64 9.3 (this host)
pkg_add: Warning: package `curl-8.8.0nb1' was built for a platform:
pkg_add: NetBSD/x86_64 9.0 (pkg) vs. NetBSD/x86_64 9.3 (this host)
pkg_add: A different version of curl-8.8.0nb1 is already installed: curl-8.7.1
pkg_add: Can't install dependency curl>=8.7.1nb1
pkg_add: Expected dependency curl>=8.7.1nb1 still missing
pkg_add: Can't install dependency git-base>=2.45.2
pkg_add: Expected dependency git-base>=2.45.2 still missing
pkg_add: 1 package addition failed
```

Specifically, `pkg_add: NetBSD/x86_64 9.0 (pkg) vs. NetBSD/x86_64 9.3 (this host)`
you can see that the cdn packages are for NetBSD 9.0 but the image in main is 9.3.

So, either set the NetBSD version to 9.0 or the PKG_PATH needs to be updated to have the package set compiled for the specified NetBSD version.